code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -117,22 +113,16 @@ rcd light flash thingy on matter drain
 	if(C)
 		C.post_status("alert", "lockdown")
 
-	src.verbs += /mob/living/silicon/ai/proc/disablelockdown
-	src << "<span class = 'warning'>Lockdown Initiated.</span>"
-	malf_cooldown = 1
-	spawn(30)
-	malf_cooldown = 0
+	verbs -= /mob/living/silicon/ai/proc/lockdown
+	command_announcement.Announce("Hostile runtime detected in door controllers. Isolation Lockdown protocols are now in effect. Please remain calm.","Network Alert:", 'sound/misc/notice1.ogg')
+	src << "<span class = 'warning'>Lockdown Initiated. Network reset in 90 seconds.</span>"
+	spawn(900) //90 Seconds.
+		disablelockdown() //Reset the lockdown after 90 seconds.

```

Comment:
if the AI is destroyed in those 90 seconds, this will runtime. Entirely possible with MALF.